platform/x11/context_gl_x11.cpp

```diff
@@ -191,7 +191,8 @@ Error ContextGL_X11::initialize() {
 
 	swa.colormap = XCreateColormap(x11_display, RootWindow(x11_display, vi->screen), vi->visual, AllocNone);
 	x11_window = XCreateWindow(x11_display, RootWindow(x11_display, vi->screen), 0, 0, OS::get_singleton()->get_video_mode().width, OS::get_singleton()->get_video_mode().height, 0, vi->depth, InputOutput, vi->visual, valuemask, &swa);
-
+	XStoreName(x11_display, x11_window, "Godot Engine");
+	
```

Comment:
Not sure why GitHub doesn't make it selectable now, but there's a trailing tab here that should be removed.